public/app/features/plugins/plugin_loader.ts

```diff
@@ -190,7 +178,12 @@ export async function importPluginModule(path: string): Promise<any> {
       return Promise.resolve(builtIn);
     }
   }
-  return grafanaRuntime.SystemJS.import(path);
+
+  const defaultExtension = path.endsWith('.js') ? '' : '.js';
+  const baseURL = path.startsWith('/') ? '' : '/public/';
+  const cacheBuster = `?_cache=${Date.now()}`;
```

Comment:
Careful with this cache buster!  this will have a new URL every request.  The original code sets the value in the root scope so it is set only once (not at each request)